library.cpp

```diff
@@ -137,13 +137,10 @@ namespace /*anon*/ {
 
         /// Send message to remote websocket server
         /// \param data to be sent
-        void send_message(std::wstring const& data)
+        void send_message(std::wstring data)
         {
-            VERBOSE(L"Sending message: " << data);
-
-            const std::string to_send = utf8_encode(data); // BIG OOPS! lifetime ends at function exit
-            ws_.async_write(net::buffer(to_send),
-                            beast::bind_front_handler(&Session::on_write, shared_from_this()));
+            post(ws_.get_executor(),
```

Comment:
added the missed spot - hopefully it was already clear from the analogous change to `disconnect`/`do_disconnect` :)

Reply:
Actually, it wasn't. Hence my questions. It makes more sense now, thanks!